server/mediaserver.go

```diff
@@ -121,14 +121,11 @@ type authWebhookResponse struct {
 	PreviousSessions []string `json:"previousSessions"`
 	Detection        struct {
 		// Run detection on 1/freq segments
-		Freq                       uint `json:"freq"`
-		SceneClassificationProfile struct {
-			SampleRate uint `json:"sampleRate"`
-			Classes    []struct {
-				ID   int    `json:"id"`
-				Name string `json:"name"`
-			} `json:"classes"`
-		} `json:"sceneClassificationProfile"`
+		Freq                uint `json:"freq"`
+		SampleRate          uint `json:"sampleRate"`
+		SceneClassification []struct {
+			ClassName string `json:"name"`
```

Comment:
```suggestion
			Name string `json:"name"`
```

^ To match the JSON field?